attr_default.gemspec

```diff
@@ -1,12 +1,16 @@
 require File.expand_path('../lib/attr_default/version', __FILE__)
 
 Gem::Specification.new do |gem|
-  gem.authors       = ["Colin Kelley", "Nick Burwell"]
-  gem.email         = ["colindkelley@gmail.com"]
+  gem.authors       = ['Invoca development']
+  gem.email         = ['development@invoca.com']
   gem.description   = %q{Dynamic Ruby defaults for ActiveRecord attributes}
   gem.summary       = %q{Dynamic Ruby defaults for ActiveRecord attributes. These are lazy evaluated just in time: when first accessed, or just before validation or save. This allows dynamic defaults to depend on attributes that are assigned after initialization, or on other dynamic defaults.}
   gem.homepage      = "https://github.com/RingRevenue/attr_default"
 
+  gem.metadata = {
```

Comment:
This overrides the entire `metadata` hash. It's best to set only the value you need:

```ruby
gem.metadata['allowed_push_host'] = "https://rubygems.org"
```

Reply:
This notation was something that Colin said he'd preferred. Thoughts?
https://github.com/Invoca/invoca-utils/pull/8#discussion_r388598350

Reply:
Right, the entire hash is right here in front of us. There is no other code to set it AFAIK. @dtognazzini do you know of any?

Reply:
@dtognazzini What do you think?

Reply:
My concern was about overwriting the entire hash. Turns out that's fine and somewhat recommended in the Rubygems documentation:
https://guides.rubygems.org/specification-reference/#metadata

Reply:
Cool, I like it!